src/ws/session-utils.c

```diff
@@ -225,7 +225,7 @@ write_control_end (void)
   assert (auth_msg_size > 0);
   assert (auth_msg != NULL);
 
-  if (cockpit_frame_write (STDOUT_FILENO, (unsigned char *)auth_msg, auth_msg_size) < 0)
+  if (cockpit_frame_write (STDIN_FILENO, (unsigned char *)auth_msg, auth_msg_size) < 0)
```

Comment:
This looks confusing now. Perhaps this should define a new constant `BRIDGE_FD`?

Reply:
added to `cockpitframe.h`